addons/binding/org.openhab.binding.heos/README.md

```diff
@@ -0,0 +1,420 @@
+| Inputs            | String        | The input to be switched to. Input values from HEOS protocol          |
+| PlayUrl           | String        | Plays a media file located at the URL                                 |
+| Shuffle           | Switch        | Switches shuffle ON or OFF                                            |
+| RepeatMode        | String        | Defines the repeat mode: Inputs are: "One" ; "All" or "Off"           |
+
+#### Inputs depending on Player type (Date 12.02.2017):
+
+| Input names   |
+|-------------- |
+| aux_in_1      |
+| aux_in_2      |
+| aux_in_3      |
+| aux_in_4      |
+| aux1          |
+| aux2          |
+| aux3          |
+| aux4          |
+| aux5          |
+| aux6          |
+| aux7          |
+| line_in_1     |
+| line_in_2     |
+| line_in_3     |
+| line_in_4     |
+| coax_in_1     |
+| coax_in_2     |
+| optical_in_1  |
+| optical_in_2  |
+| hdmi_in_1     |
+| hdmi_arc_1    |
+| cable_sat     |
+| dvd           |
+| bluray        |
+| game          |
+| mediaplayer   |
+| cd            |
+| tuner         |
+| hdradio       |
+| tvaudio       |
+| phono         |
+
+An actual list can be found within the HEOS CLI protocol which can be found [here](http://rn.dmglobal.com/euheos/HEOS_CLI_ProtocolSpecification.pdf).
+
+### The Bridge provides the following channels:
+
+| Channel ID            | Item Type     | Description                                                                                                                                               |
+|---------------------- |-----------    |--------------------------------------------------------------------------------------------------------------------------------------------------------   |
+| Reboot                | Switch        | Reboot the whole HEOS System. Can be used if you get in trouble with the system                                                                           |
+| DynamicGroupHandling  | Switch        | If this option id activated the system automatically removes groups if they are ungrouped. Only works if the group is added via an UI.                    |
+| BuildGroup            | Switch        | Is used to define a group. The player which shall be grouped has to be selected first. If Switch is then activated the group is build.                    |
+| Playlists             | String        | Plays a playlist on the prior selected Player Channel (see below) Playlists are identified by numbers. List can be found in the HEOS App                  |
+| RawCommand            | String        | A channel where every HEOS CLI command can be send to.                                                                                                    |
+| PlayUrl               | String        | Plays a media file located at the URL. First select the player channel where the stram shall be played. Then send the stream via the Play URL channel.    |
```

Comment:
Why is this channel on the bridge, can you explain your description?

Do you have a single bridge above all the HEOS devices or is it on bridge per device?

Reply:
This is an old channel. First I only had it on the bridge. Now,each player has an PlayURL channel. This one at the bridge can be removed. 
There is one bridge for the whole HEOS system which controls the system.

Reply:
So I checked it. It is possible to play and URL via the bridge by selecting the player channel first or by using the channel at the player and groups. 
Maybe it would be god to have only one way. 
wdyt?

Reply:
Still to be removed?

Reply:
Yes